packages/ui/src/models/camel/camel-route-resource.test.ts

```diff
@@ -209,17 +209,9 @@ describe('CamelRouteResource', () => {
       [{ anotherUnknownContent: {} }],
       [{}],
     ])('should not throw error when calling: %s', (json) => {
-      const resource = new CamelRouteResource(json);
+      const resource = new CamelRouteResource(JSON.stringify(json));
       const firstEntity = resource.getVisualEntities()[0] ?? resource.getEntities()[0];
       expect(firstEntity.toJSON()).not.toBeUndefined();
     });
   });
-
-  describe('comments', () => {
-    it('should set and get comments', () => {
-      const resource = new CamelRouteResource();
-      resource.setComments(['a', 'b']);
-      expect(resource.getComments()).toEqual(['a', 'b']);
-    });
-  });
```

Comment:
Would it be possible to instead of removing the test, prepare a YAML with some comments and checking if they are preserved?